common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4282,6 +4292,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("3s")
 
+  val CLIENT_RPC_RETRY_WAIT: ConfigEntry[Long] =
+    buildConf("celeborn.client.rpc.retryWait")
+      .categories("client")
+      .version("0.6.0")
```

Comment:
> 0.5.4

@turboFei Sorry deleted your version change comment by mistake, thinking it is my draft comment. Commenting this for posterity.

Reply:
Fixed the version.